src/Arcus.Scripting.DevOps/Scripts/Save-AzDevOpsBuild.ps1

```diff
@@ -1,28 +1,26 @@
 param(
    [Parameter(Mandatory = $true)][string] $ProjectId = $(throw "ProjectId is required"),
-   [Parameter(Mandatory = $true)][string] $BuildId = $(throw "BuildId is required")   
+   [Parameter(Mandatory = $true)][string] $BuildId = $(throw "BuildId is required")
 )
 
-$retentionPayload = @{        
+$retentionPayload = @{
   keepforever='true'
 }
 
 $requestBody = $retentionPayload | ConvertTo-Json -Depth 1 -Compress
 
 $collectionUri = $env:SYSTEM_COLLECTIONURI
-
-if( $collectionUri.EndsWith('/') -eq $False ){
+if ($collectionUri.EndsWith('/') -eq $false) {
   $collectionUri = $collectionUri + '/'
 }
 
 $requestUri = "$collectionUri" + "$ProjectId/_apis/build/builds/" + $BuildId + "?api-version=6.0"
 
-Write-Verbose "Saving AzDevOps build with buildid $BuildId in project $ProjectId by posting $requestBody to $requestUri"
-
-$response = Invoke-WebRequest -Uri $requestUri -Method Patch -Body $requestBody -ContentType "application/json" -Headers @{ Authorization = "Bearer $env:SYSTEM_ACCESSTOKEN" }      
+Write-Verbose "Saving Azure DevOps build with build ID $BuildId in project $ProjectId by posting $requestBody to $requestUri"
+$response = Invoke-WebRequest -Uri $requestUri -Method Patch -Body $requestBody -ContentType "application/json" -Headers @{ Authorization = "Bearer $env:SYSTEM_ACCESSTOKEN" }
 
-if( $response.StatusCode -ne 200 ) {
-    Throw "Unable to retain build indefinetely. API request returned statuscode $($response.StatusCode)"    
+if ($response.StatusCode -ne 200) {
+    throw "Unable to retain build indefinetely. API request returned statuscode $($response.StatusCode)"
```

Comment:
Why not `Throw` ?  Isn't the coding style in PS `Throw` ?

Reply:
I've seen both practices around. I'm streamlining it with the other scripts in this repo. If you think we should re-think this, maybe we should start a discussion/issue to talk about style there. For the scope of this PR, it's best to streamline it first.

Reply:
If `throw` is used in other scripts, we should indeed do the same here.  Consistency is key.